subworkflows/local/compute_gene_scores_heatmap/main.nf

```diff
@@ -9,12 +9,12 @@ include { COMPUTE_GENE_SCORES     } from '../../../modules/local/compute_gene_sc
 include { CREATE_GENE_HEATMAP     } from '../../../modules/local/create_gene_heatmap'
 
 workflow COMPUTE_GENE_SCORES_HEATMAP {
+
     take:
     normalized_counts          // channel: [ meta, normalized_counts.tsv ]
     annotated_endo_data        // channel: [ meta, annotated_endo_data.tsv ]
     ch_gene_score_yaml         // channel: [ file(gene_score_yaml.yaml) ]
     ch_heatmap_genes_to_filter // channel: [ file(heatmap_genes_to_filter.yaml) ]
-    skip_heatmap               // boolean
```

Comment:
Please revert this change, the subworkflow shouldn't get params being passed directly into it, that is why we are passing the params.skip_heatmap